exercises/forth/forth_test.py

```diff
@@ -0,0 +1,234 @@
+import unittest
+
+from forth import evaluate
+
+
+class ForthAdditionTest(unittest.TestCase):
```

Comment:
Are these test cases adopted from https://github.com/exercism/problem-specifications/blob/master/exercises/forth/canonical-data.json?
If so, please state that in the comment denoting version as well

Reply:
Yes, they were adapted from the canonical data. Comment with test data version has been added.

Reply:
Don't forget that `from example import ...` needs to be changed back to `from forth import ...`.